helix-term/src/compositor.rs

```diff
@@ -64,10 +67,20 @@ pub trait Component: Any + AnyComponent {
 }
 
 use anyhow::Context as AnyhowContext;
+use tui::backend::Backend;
+
+#[cfg(not(feature = "integration"))]
+use tui::backend::CrosstermBackend;
+
+#[cfg(not(feature = "integration"))]
 use std::io::stdout;
-use tui::backend::{Backend, CrosstermBackend};
+
+#[cfg(not(feature = "integration"))]
 type Terminal = tui::terminal::Terminal<CrosstermBackend<std::io::Stdout>>;
 
+#[cfg(feature = "integration")]
+type Terminal = tui::terminal::Terminal<TestBackend>;
+
```

Comment:
Oh interesting. Would that goal be to get rid of the feature flags? I think we'd still have to have them, since we're still building an Application.

Or are you just asking me to do it to get the code change into master?

Reply:
It's part of an upcoming refactor and I think it would simplify the amount of `cfg` clauses you need

Reply:
I'm sorry, I tried, but the code seems to be built on top of other bigger breaking API changes, like adding a new `RenderContext`. It's difficult to take just those changes without the rest of the branch.